pkg/kev/config/schema.go

```diff
@@ -67,6 +67,82 @@ var ServicesSchema = map[string]interface{}{
 		LabelWorkloadReplicas,
 		LabelWorkloadLivenessProbeType,
 	},
+	"allOf": []map[string]interface{}{
+		// Liveness probe
+		{
+			"oneOf": []map[string]interface{}{
+				{
+					"properties": map[string]interface{}{
+						LabelWorkloadLivenessProbeType: map[string]interface{}{
+							"const": "http",
+						},
+					},
+					"required": []string{LabelWorkloadLivenessProbeHTTPPath, LabelWorkloadLivenessProbeHTTPPort},
+				},
+				{
+					"properties": map[string]interface{}{
+						LabelWorkloadLivenessProbeType: map[string]interface{}{
+							"const": "tcp",
+						},
+					},
+					"required": []string{LabelWorkloadLivenessProbeTCPPort},
+				},
+				{
+					"properties": map[string]interface{}{
+						LabelWorkloadLivenessProbeType: map[string]interface{}{
+							"const": "command",
+						},
+					},
+					"required": []string{LabelWorkloadLivenessProbeCommand},
+				},
+				// This acts as a catch all when the required properties are not provided, should come in the end
+				// otherwise instead of matching that "command" type needs a properties the error says the value is not "none".
```

Comment:
slight rewording of this comment would be beneficial

Reply:
do you have a specific suggestion besides fixing the typo?

Reply:
I think just a simple rephrase would help to make it clearer to follow.

Reply:
I removed the why and just left the what to do 😆